src/lib/token_list_gen.ts

```diff
@@ -429,6 +438,28 @@ export const generateFullList = async () => {
   const etherscanData = arbListtoEtherscanList(tokenData);
   return etherscanData;
 };
+export const generateFullListFormatted = async () => {
```

Comment:
Looks similar to `generateFullList`, the difference being the call to `arbListtoEtherscanList` in `generateFullList`, can we extract the common logic to a separate function? Or maybe one function with parameter? `generateFullList({ format }: { format: true })`

Reply:
They are similar, but I'd rather just keep them separate since the common logic is minimal / about as long as the different logic